index.html

```diff
@@ -27,46 +27,227 @@
+        </div>
+        <div class="social">
+          <div class="social-text"><h4>LET'S CONNECT</h4></div>
+          <div class="social-icon">
+            <ul>
+              <li>
+                <a href="https://github.com/QueenterJuma"
+                  ><i class="fa-brands fa-github"></i
+                ></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/queenter-juma-54a0b1222/"
+                  ><i class="fa-brands fa-linkedin"></i
+                ></a>
+              </li>
+              <li>
+                <a href="https://twitter.com/QueenJumah"
+                  ><i class="fa-brands fa-twitter"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-solid fa-m"></i></a>
+              </li>
+              <li>
+                <a href="#"><img src="./image/vector.png" alt="" /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </sectiion>
+    <section class="card">
+      <div class="card-container">
+        <div class="card-image one">
+          <img src="./image/pic1.png" alt="img" />
+        </div>
+        <div class="card-content">
+          <h3>Tonic</h3>
+          <ul class="institution">
+            <li><a href="#">CANOPY</a></li>
+            <div class="dot"></div>
+            <li><a href="#">Back End Dev</a></li>
+            <div class="dot"></div>
+            <li><a href="#">2015</a></li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="tech">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <button>See Project</button>
+        </div>
       </div>
-      <div class="social">
-        <div class="social-text"><h4>LET'S CONNECT</h4></div>
-        <div class="social-icon">
-          <ul>
-            <li>
-              <a href="https://github.com/QueenterJuma"
-                ><i class="fa-brands fa-github"></i
-              ></a>
+      <div class="card-container">
+        <div class="card-image two">
+          <img src="./image/pic2.png" alt="img" />
+        </div>
+        <div class="card-content">
+          <h3>Multipost Stories</h3>
+          <ul class="institution">
+            <li><a href="#">CANOPY</a></li>
+            <div class="dot"></div>
+            <li><a href="#">Back End Dev</a></li>
+            <div class="dot"></div>
+            <li><a href="#">2015</a></li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="tech">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
```

Comment:
- [ ] [ OPTIONAL ] These list items should just be texts to show the technologies used to develop the project, and they are not supposed to redirect the user. So they are not supposed to be links. Please consider replacing the `a` element with a `p` element. And also make the same changes in all the other work cards.